src/h5web/vis-packs/core/shared/TooltipMesh.tsx

```diff
@@ -74,10 +75,9 @@ function TooltipMesh(props: Props) {
 
   return (
     <>
-      <mesh {...{ onPointerMove, onPointerOut, onPointerDown, onPointerUp }}>
+      <Mesh {...{ onPointerMove, onPointerOut, onPointerDown, onPointerUp }}>
         <meshBasicMaterial opacity={0} transparent />
-        <planeGeometry args={[width, height]} />
-      </mesh>
+      </Mesh>
```

Comment:
Now using `Mesh` that has the sets the geometry to the vis size and places the `mesh` at the right position